tests/integration/package/test_package_command.py

```diff
@@ -440,6 +440,39 @@ def test_package_with_resolve_s3(self, template_file):
                 process_stdout,
             )
 
+    @parameterized.expand(["1", "0"])
+    def test_package_with_no_progressbar(self, no_progressbar):
+        template_path = self.test_data_path.joinpath("aws-serverless-function.yaml")
+        s3_prefix = "integ_test_prefix"
+
+        with tempfile.NamedTemporaryFile(delete=False) as output_template:
+            command_list = self.get_command_list(
+                template_file=template_path,
+                s3_prefix=s3_prefix,
+                output_template_file=output_template.name,
+                force_upload=True,
+                no_progressbar=bool(int(no_progressbar)),
```

Comment:
you can just pass in True, False instead of "1", "0"

Reply:
@sriram-mv Changed this in the latest commit